### PR TITLE
[BODS-3532] updated in-scope out-of-scope logic for lta report

### PR DESCRIPTION
**Pull request by amitkpmguk**

### CODE REVIEW

File: transit_odp/browse/common.py [hunk 1]
```diff
@@ -1,18 +1,22 @@
...
+import pandas as pd
+
 from transit_odp.naptan.models import AdminArea
-from transit_odp.otc.models import Service
 from transit_odp.organisation.constants import TravelineRegions
-from typing import Dict
+from transit_odp.otc.models import Service
 
 
 def get_all_naptan_atco_map() -> Dict[str, str]:
```
Comment (ankitmahajan1) [inline transit_odp/browse/common.py:10]: change the return type for this function 
and we can rename the function as it not returning the map or dict

File: transit_odp/browse/common.py [hunk 2]
```diff
@@ -73,6 +77,9 @@ def get_service_traveline_regions(ui_ltas):
     return "|".join(
         [
             str(TravelineRegions(admin_area.traveline_region_id).label)
-            for admin_area in AdminArea.objects.filter(ui_lta__in=ui_ltas).all()
+            for admin_area in AdminArea.objects.filter(ui_lta__in=ui_ltas)
+            .order_by("traveline_region_id")
```
Comment (ankitmahajan1) [inline transit_odp/browse/common.py:81]: we can apply order_by after distinct so less records need to be operated on

Comment (amitkpmguk) [reply transit_odp/browse/common.py]: Updated

File: transit_odp/browse/common.py (hunk 1 above)
Comment (amitkpmguk) [reply transit_odp/browse/common.py]: Updated

**Review by ankitmahajan1 — APPROVED** [review]
approved

**Pull request merged**